unstructured/partition/msg.py

```diff
@@ -1,239 +1,17 @@
-from __future__ import annotations
-
-import os
-import tempfile
-from typing import IO, Any, Callable, Optional
+# pyright: reportPrivateUsage=false
```

Comment:
This suppresses a type-error on the publication of `_AttachmentPartitioner` and `_MsgPartitioner` that are private classes. The type checker (`pyright`) would otherwise rightly complain "hey, you shouldn't be publishing private classes or methods". This will go away in the next PR when we rename `partition/new_msg.py` to `partition/msg.py` and get rid of this transitional version of `msg.py`.

This pragma sometimes legitimately appears in a test module because we're testing private classes or methods, but generally should not appear in a code module. The only reason it does here is because I did this "redirect old->new" thing to keep the diffs from being scrambled.

Reply:
Gotcha, thanks!